Firestore/Example/Tests/SpecTests/FSTMockDatastore.m

```diff
@@ -90,10 +91,14 @@ - (BOOL)isStarted {
   return self.open;
 }
 
-- (void)handleStreamOpen {
+- (void)notifyStreamOpen {
   [self.delegate watchStreamDidOpen];
 }
 
+- (void)notifyStreamInterrupted:(NSError *_Nullable)error {
```

Comment:
Naming: `notifyStreamInterruptedWithError:` Also prefer the `nullable` attribute to the `_Nullable` qualifier.

This should be

```objc
- (void)notifyStreamInterruptedWithError:(nullable NSError *)error
```